xbmc/dbwrappers/dataset.h

```diff
@@ -469,8 +471,13 @@ class Dataset  {
 /*clear delete_sql*/
   void clear_delete_sql();
 
-/*get value of select_sql*/
- const char *get_select_sql();
+  /* size of insert_sql*/
+  size_t insert_sql_count();
+  /* size of delete_sql*/
+  size_t delete_sql_count();
+
+  /*get value of select_sql*/
+  const char* get_select_sql();
```

Comment:
Is there a reason to use this formatting for the function name? Why not `GetSelectSql()`?

Reply:
See my previous reply.